src/test/java/org/openmrs/contrib/qaframework/page/AllergiesPage.java

```diff
@@ -0,0 +1,21 @@
+package org.openmrs.contrib.qaframework.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+
+public class AllergiesPage extends Page {
+	public static final By EDITALLERGIES = By.id("allergyui-editAllergies");
+
```

Comment:
please add more of the actions expected off this page such as edit, delete, add new etc